### ISSUE TITLE
Possible to 'fake' first-published' time

### ISSUE THREAD

**Issue by runarmyklebust** [enonic/app-contentstudio#2220]
If one is setting the publish-from back in time, this value will be stored as the "first published" - time.
From a publisher POV this means that this field cannot be trusted, I think  it should be set to the actual time of publish, not what the user selects?

**Comment by alansemenov** [comment]
Ok, so your suggestion is basically to not allow setting `Published From` field to be in the past **from the UI**. It still has to be allowed via API for import/migration of items published in the past.

We'll discuss this.

**Comment by runarmyklebust** [comment]
yup; I guess that is a solution. At the moment, it is "very easy" (according to 'my' users :-) to by accident set the date to something in the past.

**Comment by alansemenov** [comment]
Not allowing "Publish from" to be in the past might be inconvenient if customers want to indicate when the article was first published on their old website, for example.
But "first publish" should always be the exact date/time of publishing, I think.

**Comment by alansemenov** [comment]
We will check with customers what they are actually using the "First Publish" field for.

**Comment by sigdestad** [comment]
Yeah, we absolutely need to check this one. I know from collicare now, they have created their content, and set "publish-from" back in time to reflect when stuff was initially pubished. But
A) What do they get when they then publish in publish.first
b) What if they change this later?
Etc.   

So, need to step very carefully here. Basically I think publish.first is the way to go if you want to sort content from when it was published. If you need to set this based on import then we do it back-end, not editorially.

Also, what field are we sorting on when you choose sort order "publish time"?

**Comment by sigdestad** [comment]
Usage: Publish-first is set based on publish-from date on the first publish, this is used for consistently sorting content over time, as publish-from may change later.

Improvement suggestion:
* Better visualisation when publish-from is set "back in time"
* Default vaule will be "Now"
* Also, only users with "Editor" role for the project will be allowed to set publish-first back in time. This is frequently used when migrating content manually
